cfgov/v1/documents.py

```diff
@@ -34,6 +54,8 @@ class FilterablePagesDocument(Document):
     statuses = fields.KeywordField()
     initial_filing_date = fields.DateField()
     model_class = fields.KeywordField()
+    content = fields.TextField()
+    preview_description = fields.TextField()
```

Comment:
Is it deliberate that only these fields are included in the search? Some of the filterable pages include other fields; for example, DocumentDetailPage, EventPage, and LearnPage include a [`header`](https://github.com/cfpb/consumerfinance.gov/blob/12bcc7de85ab15ca9c9961ff7724990e745c3680/cfgov/v1/models/learn_page.py#L30-L34) field, EventPage includes several other [body fields](https://github.com/cfpb/consumerfinance.gov/blob/12bcc7de85ab15ca9c9961ff7724990e745c3680/cfgov/v1/models/learn_page.py#L178-L182), etc.

Reply:
These we're the fields specified by Sonna as the initial ask